### ISSUE TITLE
Specific listen port for specific service

### ISSUE THREAD

**Issue by lemonshow** [kubernetes-sigs/aws-alb-ingress-controller#575]
Is this supported? We have a similar use case.
came across https://github.com/kubernetes-sigs/aws-alb-ingress-controller/issues/135

Annotation: alb.ingress.kubernetes.io/listen-ports: '[{"HTTP": 3033}, {"HTTP": 3001, "HTTPS": 3002}]'

or with service-listen-ports

/Service-1 (NodePort/Target Group 123):
Http Port 3033

/Service-2 (NodePort/Target Group 234):
Http Port 3001
Https Port 3002

From the above discussion, seems still open.

**Comment by M00nF1sh** [comment]
@lemonshow 
We do aware of it but hasn't put it into roadmap yet.
I don't know whether multiple listener port fits into k8s's ingress. If we want to have such feature, the best way might be ingress itself adds `ingressPort` support themself, like the servicePort. Otherwise, supporting such feature seems hacky...

From my point-of-view, If you are going to have different rules for different port(other than plain http->https redirection), maybe it should be two ingress instead(if we supported aggregate multiple ingress to share a single alb somehow).

In conclusion, we don't know the correct way to support such feature now, and will implemented it if the community can reach agreement on the correct way to support it.

**Comment by lemonshow** [comment]
https://github.com/kubernetes-sigs/aws-alb-ingress-controller/issues/135

#135 is still open, thought you have a plan for it.

We have for example ingress with below,
    alb.ingress.kubernetes.io/listen-ports: '[{"HTTP": 9000, "HTTPS": 9001},{"HTTP": 9099}]'
What we got are with 3 Listeners at 3 ports, they all point to the same TG. The reason we need another HTTP port 9099 is that one user might need to use different port for one of the service API.
So there are redundancy. Do you see any issue?

**Comment by lemonshow** [comment]
Beside, I can do this with ALB console to let my port 9099 point to a different forwarding rule sets with one of the service TG.
